### DISCUSSION TITLE
I can't summarize, can you help me?

### DISCUSSION THREAD

**Discussion by rafassotero** [MuiseDestiny/zotero-gpt#150] [Q&A]
I managed to register the key, but when I execute any function on the article it keeps running and i don't know if it was executed nor where the saved file is. what am i doing wrong?

**Comment by MuiseDestiny** [comment]
Can you provide your `/report` screenshot?

![image](https://github.com/MuiseDestiny/zotero-gpt/assets/51939531/15e7fe09-4447-4c95-97ba-f95220039a53)
